shared/src/main/scala/com/raquo/domtypes/generic/defs/tags/SvgTags.scala

```diff
@@ -0,0 +1,733 @@
+  
+  /**
+   * The foreignObject element allows for inclusion of a foreign XML namespace
+   * which has its graphical content drawn by a different user agent. The
+   * included foreign graphical content is subject to SVG transformations and
+   * compositing.
+   *
+   * MDN
+   */
+  lazy val foreignObject:T[SvgElement] = tag("foreignObject")
+  
+  /**
+   * The g element is a container used to group objects. Transformations applied
+   * to the g element are performed on all of its child elements. Attributes
+   * applied are inherited by child elements. In addition, it can be used to
+   * define complex objects that can later be referenced with the use element.
+   *
+   * MDN
+   */
+  lazy val g:T[SvgG] = tag("g")
+  
+  /**
+   * A glyph defines a single glyph in an SVG font.
+   *
+   * MDN
+   */
+  lazy val glyph:T[SvgElement] = tag("glyph")
+  
+  /**
+   * The glyphRef element provides a single possible glyph to the referencing
+   * altGlyph substitution.
+   *
+   * MDN
+   */
+  lazy val glyphRef:T[SvgElement] = tag("glyphRef")
+  
+  /**
+   * The horizontal distance between two glyphs can be fine-tweaked with an
+   * hkern Element. This process is known as Kerning.
+   *
+   * MDN
+   */
+  lazy val hkern:T[SvgElement] = tag("hkern")
+  
+  /**
+   * The SVG Image Element (image) allows a raster image into be included in
+   * an SVG document.
+   *
+   * MDN
+   */
+  lazy val image:T[SvgImage] = tag("image")
+  
+  /**
+   * The line element is an SVG basic shape, used to create a line connecting
+   * two points.
+   *
+   * MDN
+   */
+  lazy val line:T[SvgLine] = tag("line")
+  
+  /**
+   * linearGradient lets authors define linear gradients to fill or stroke
+   * graphical elements.
+   *
+   * MDN
+   */
+  lazy val linearGradient:T[SvgLinearGradient] = tag("linearGradient")
+  
+  /**
+   * The marker element defines the graphics that is to be used for drawing
+   * arrowheads or polymarkers on a given path, line, polyline or
+   * polygon element.
+   *
+   * MDN
+   */
+  lazy val marker:T[SvgMarker] = tag("marker")
+  
+  /**
+   * In SVG, you can specify that any other graphics object or g element can
+   * be used as an alpha mask for compositing the current object into the
+   * background. A mask is defined with the mask element. A mask is
+   * used/referenced using the mask property.
+   *
+   * MDN
+   */
+  lazy val mask:T[SvgMask] = tag("mask")
+  
+  /**
+   * Metadata is structured data about data. Metadata which is included with SVG
+   * content should be specified within metadata elements. The contents of the
+   * metadata should be elements from other XML namespaces such as RDF, FOAF,
+   * etc.
+   *
+   * MDN
+   */
+  lazy val metadata:T[SvgMetadata] = tag("metadata")
+  
+  /**
+   * The missing-glyph's content is rendered, if for a given character the font
+   * doesn't define an appropriate glyph.
+   *
+   * MDN
+   */
+  lazy val `missing-glyph`:T[SvgElement] = tag("missing-glyph")
```

Comment:
We should not be using backticks. Tags with dashed should be camelCased instead. (same for other tags)

Reply:
That's true,gonna do that later with regex after initial test